dotcom-rendering/src/web/lib/useApi.tsx

```diff
@@ -27,12 +27,14 @@ interface ApiResponse<T> {
  * returns { loading, error, data }
  * @param {String} url - The url to fetch
  * @param {SWRConfiguration} options - The SWR config object - https://swr.vercel.app/docs/options
+ * @param {RequestInit} init - The fetch init object - https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#supplying_request_options
  * */
 export const useApi = <T,>(
 	url: string,
 	options?: SWRConfiguration,
+	init?: RequestInit,
 ): ApiResponse<T> => {
```

Comment:
Maybe using `Data` instead of `T` this type is more explicit, and `extends unkown` is better than the odd dangling comma to prevent TSX from tripping up?

```suggestion
export const useApi = <Data extends unknown>(
	url: string,
	options?: SWRConfiguration,
	init?: RequestInit,
): ApiResponse<Data> => {
```